src/predefined-variables.ts

```diff
@@ -37,12 +43,16 @@ export function init ({gitData, argv}: PredefinedVariablesOpts): {[name: string]
         CI_SERVER_FQDN: `${gitData.remote.host}`,
         CI_SERVER_HOST: `${gitData.remote.host}`,
         CI_SERVER_PORT: `${gitData.remote.port}`,
-        CI_SERVER_URL: `https://${gitData.remote.host}:443`,
+        CI_SERVER_URL: CI_SERVER_URL,
```

Comment:
I apologize for stupid questions (I'm not a TypeScript developer), but isn't this self-referential? I would expect this to be `${gitData.remote.host}:${gitData.remote.port}`

Reply:
Nevermind. I realized what's going on here.